changelog/1.36.0.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "1.36.0"
+description: "Released on TBD"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.36.0
+
+### Features ✨
+
+- infra: Add sessionAffinity to coderd service.
+- infra: Allow older verions of curl for self-contained builds.
+- infra: Include coder-cli asset to coderd-run-dev for self-contained builds.
```

Comment:
Development only.  I don't think this should be included in customer-facing release notes.

Reply:
Oof, I misunderstood the change.  Will remove!